src/App.tsx

```diff
@@ -1,39 +1,53 @@
+/* eslint-disable no-console */
 import React from 'react';
 import './App.scss';
+import Clock from './Clock';
 
-function getRandomName(): string {
-  const value = Date.now().toString().slice(-4);
+// function getRandomName(): string {
+//   const value = Date.now().toString().slice(-4);
 
-  return `Clock-${value}`;
-}
+//   return `Clock-${value}`;
+// }
+
+// This code starts a timer
+// const timerId = window.setInterval(() => {
+//   clockName = getRandomName();
+// }, 3300);
 
-export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+// this code stops the timer
+// window.clearInterval(timerId);
+
```

Comment:
You can remove not used code :) Try to clean commented code and don't keep comments if it's not needed